server/build.gradle.kts

```diff
@@ -51,4 +51,6 @@ tasks.withType<BootRun> {
 
 tasks.withType<Test> {
     useJUnitPlatform()
+    // excluded from CI until MongoDB testcontainers is fixed
+    exclude("**/integration/**")
 }
```

Comment:
don't exclude all in gradle.
Use @Disabled on IT.java. E.g. the RegisterServerIT does run.

Reply:
Only the first IT runs successfully (selected randomly by JUnit), the next ones will fail to run because the MongoDB container will not release port 27017 and the container is not reused. So it doesn't make sense to disable all integration tests except one. I prefer to disable all of them in one place rather then manage them individually.